pkg/k8scontext/context.go

```diff
@@ -424,5 +424,6 @@ func (c *Context) Stop() {
 }
 
 func isIngressApplicationGateway(ingress *v1beta1.Ingress) bool {
-	return annotations.IsApplicationGatewayIngress(ingress)
+	val, _ := annotations.IsApplicationGatewayIngress(ingress)
```

Comment:
Probably not a bad idea to `glog.Error` here when the if `annotations.IsApplicationGatewayIngress` returns a non-nil error.

Reply:
Will add in a following PR!